domains/shahin.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Describe the use of this subdomain",
+    "repo": "https://github.com/Shahinn0x/Portfolio",
+    "owner": {
+        "username": "Shahinn0x",
+        "email": "shahin14623@gmail.com"
+    },
+    "record": {
+        "CNAME": "Shahinn0x.github.io"
```

your website is incomplete
```suggestion
        "CNAME": "shahinn0x.github.io"
```